twingate/client_resource.go

```diff
@@ -410,8 +410,8 @@ func (client *Client) readResourcesByName(ctx context.Context, name string) ([]*
 		return nil, NewAPIErrorWithID(err, "read", resourceResourceName, "All")
 	}
 
-	if len(response.Resources.Edges) == 0 {
-		return nil, NewAPIErrorWithID(ErrGraphqlResourceNotFound, "read", resourceResourceName, "All")
+	if response.Resources.Edges == nil {
```

Comment:
This is wrong. `Edges` can't be `nil`.
The previous statement was correct its just that it shouldnt raise an error but return an empty list result